framework_tvb/tvb/core/services/figure_service.py

```diff
@@ -62,22 +61,19 @@ class FigureService:
     _DEFAULT_SESSION_NAME = "Default"
     _DEFAULT_IMAGE_FILE_NAME = "snapshot."
 
-
     def __init__(self):
         self.logger = get_logger(self.__class__.__module__)
         self.file_helper = FilesHelper()
 
-
     def _write_png(self, store_path, export_data):
         img_data = base64.b64decode(export_data)                        # decode the image
-        final_image = Image.open(StringIO(img_data))                    # place it in a PIL stream
+        final_image = Image.open(BytesIO(img_data))                    # place it in a PIL stream
```

Comment:
Have you also tested this from web GUI manually, with the save figure button ?

Reply:
yes. tested